apps/mobile-wallet/package.json

```diff
@@ -48,7 +48,7 @@
     "big-integer": "^1.6.51",
     "colord": "^2.9.3",
     "dayjs": "^1.10.7",
```

Comment:
App crashed when importing a wallet...

<img width="200" alt="image" src="https://github.com/user-attachments/assets/d5a54ec7-1b78-4452-a46a-b3c71fec2648">

Reply:
It happens consistently, I tried again...

<img width="350" alt="image" src="https://github.com/user-attachments/assets/744e6a35-466b-48b2-87c2-86efe6c3ba25">

Reply:
Yeah I have it too. I wonder what made this crash, probably a deps update.